csp-mixin/azureconfig.libsonnet

```diff
@@ -14,5 +14,7 @@
     groupLabels: ['job', 'resourceGroup', 'subscriptionName'],
     instanceLabels: ['resourceName'],
     metricsSource: 'azuremonitor',
+    alertAzureVMAvailableMemoryLowThreshold: '367001600',
```

Comment:
Is it possible to determine the percentage of used/remaining memory? Usually we prefer to use a percentage, so that it scales with the VM.

Unfortunately, memory thresholds are difficult at times. For certain nodes (work stations, web servers, application nodes) high memory utilization percentage may not be desirable, but for others (caching servers, database servers) using as much memory as possible is desirable.

Reply:
I removed this alert and added another one for cpu utilization under 80% and unavailability of the VM.